tests/quarkus/pom.xml

```diff
@@ -63,80 +57,60 @@
       <groupId>org.apache.camel.quarkus</groupId>
       <artifactId>camel-quarkus-quartz</artifactId>
     </dependency>
-    <!--
-      This dependency is mandatory for enabling Camel management
-      via JMX / Hawtio.
-    -->
     <dependency>
       <groupId>org.apache.camel.quarkus</groupId>
       <artifactId>camel-quarkus-management</artifactId>
     </dependency>
-
-    <!-- To enable Camel route dumping to XML. Required for the route diagram -->
     <dependency>
       <groupId>org.apache.camel.quarkus</groupId>
       <artifactId>camel-quarkus-jaxb</artifactId>
     </dependency>
-
-    <!--
-      To enable Camel plugin debugging feature, add this dependency.
-    -->
     <dependency>
       <groupId>org.apache.camel.quarkus</groupId>
       <artifactId>camel-quarkus-debug</artifactId>
     </dependency>
-
-    <!-- Quarkus -->
+    <dependency>
+      <groupId>org.graalvm.sdk</groupId>
+      <artifactId>graal-sdk</artifactId>
+    </dependency>
```

Comment:
do we really need this? I remember it's no longer necessary but is it yet in product?